Update the documentation links

faif opened the pull request:
Avoid the confusion by pointing to the up-to-date documentation resources

faif committed:
Update the documentation links

Avoid the confusion by pointing to the up-to-date documentation resources

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/TheThingsNetwork/docs?pullRequest=121) <br/>All committers have signed the CLA.

htdvisser commented on _content/devices/lopy/index.md:
```diff
@@ -34,7 +34,7 @@ This guide will walk you through using The Things Network on the LoPy as node.
 * [Expansion Board](https://www.pycom.io/product/expansion-board/).
 
-  > Mount the LoPy with the LED (white square) [positioned on the same side](https://docs.pycom.io/lopy/lopy/tutorial/intro.html#plugging-into-the-expansion-board-and-powering-on) as the Micro-USB connector of the expansion board.
+  > Mount the LoPy with the LED (white square) [positioned on the same side](https://docs.pycom.io/pycom_esp32/pycom_esp32/getstarted.html#placement-on-the-expansion-board) as the Micro-USB connector of the expansion board.
```
Invalid link

htdvisser reviewed (changes requested):
Thanks for your changes. Please check the second link, as it seems that it was changed.

faif committed:
Fix link error and link to firmware downloader

Fix the broken link to the placement on  the expansion board. Also change the link to downloads to point to the firmware downloader
